### PR TITLE
1907-07-01

### PR DESCRIPTION
**Pull request by hcf16b**

### CODE REVIEW

**Comment by whanley** [comment]
Good start, Hannah--looks like you've done good work on the dates. But there are some structural problems. I see a red box when I look at this file. I've made some comments where I see problems. Also, you are not using the `<cb/>` tag properly. It should only appear at the start of very new column--and there are only 3 of them.

File: 1907-07-01.xml [hunk 1]
```diff
@@ -0,0 +1,1213 @@
...
+                                <cell><date when="1907-04-16">April 16</date></cell>
+                            </row>
+                        </table>
+                        <p>The S.S. Trojan Prince is now loading for <placeName>Manchester</placeName>.</p>
+                    </div>
+                    <cb n="16"/>
+                    <--!Summer Trips to Cypress-->
+                    <div/>
+                    <cb n="17"/>
+                        <body>
```
Comment (whanley) [inline 1907-07-01.xml]: delete

File: 1907-07-01.xml [hunk 2]
```diff
@@ -0,0 +1,1213 @@
...
+                                        <cell>arr.</cell>
+                                        <cell>Halfa</cell>
+                                        <cell>depart</cell>
+                                        <cell>Fri. and Mon. 6 a.m.</cell>
+                                        <cell>Sat and Wedn</cell>
+                                        <cell>noon.</cell>
+                                    </row>
+                                    <row>
+                                        <cell>Wednesdays and Saturdays</cell>
+                                        <cell/>Wednesdays and Saturdays</cell>
```
Comment (whanley) [inline 1907-07-01.xml]: use `<cell>` not `<cell/>`, which closes the cell as soon as it opens it. Similar errors below.

File: 1907-07-01.xml [hunk 3]
```diff
@@ -0,0 +1,1180 @@
...
+                                <cell>Kaffir Prince</cell>
+                                <cell>due from</cell>
+                                <cell><placeName>Manchester</placeName></cell>
+                                <cell><date when="1907-04-16">April 16</date></cell>
+                            </row>
+                        </table>
+                        <p>The S.S. Trojan Prince is now loading for <placeName>Manchester</placeName>.</p>
+                    </div>
+                    <!-- Summer Trips to Cypress -->
+                    </div>
```
Comment (whanley) [inline 1907-07-01.xml]: this `</div>` tag closes the advertising section--it should come just before the end of the page.

File: 1907-07-01.xml [hunk 4]
```diff
@@ -0,0 +1,1180 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head>No.7800 <date> 1907-07-01</date> Pages 6</head>
+                    <head> type="sub">No.7800 <measure quantity="7800">7,800</measure>] ALEXANDRIA, MONDAY, JULY 01, 1907. [SIX PAGES P.T.
+                        1. </head>
+                </div>
```
Comment (whanley) [inline 1907-07-01.xml]: the masthead should contain the two ads that appear to the left and right of the title.

File: 1907-07-01.xml [hunk 5]
```diff
@@ -0,0 +1,1180 @@
...
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head>No.7800 <date> 1907-07-01</date> Pages 6</head>
+                    <head> type="sub">No.7800 <measure quantity="7800">7,800</measure>] ALEXANDRIA, MONDAY, JULY 01, 1907. [SIX PAGES P.T.
```
Comment (whanley) [inline 1907-07-01.xml]: delete the > to the right of the word head. Delete the 7800 in the line "No.7800"

File: 1907-07-01.xml [hunk 6]
```diff
@@ -0,0 +1,1573 @@
...
+                        <head>Messageries Maritimes.</head>
+                        <p>From Alexandria</p>
+                        <table rows="12" cols="5">
+                            <head><hi rend="bold">Sailing from Alexandria in June, 1907.</hi></head>
+                            <row>
+                                <cell cols="5"><hi rend="bold">For Marseilles direct</hi></cell>
+                            </row>
+                            <row>
+                                <cell>Friday</cell>
+                                <cell><date when="1905-07-05">05 July</date></cell>
```
Comment (whanley) [inline 1907-07-01.xml]: There are still a few dates (such as this one) in 1904 1905 and 1906. Find them using `//date/@when` xpath search in oxygen, and correct them.

File: 1907-07-01.xml [hunk 7]
```diff
@@ -1170,11 +1170,404 @@
                             <!-- No template for my newspaper's version of THE Egyptian State Railways -->
                 </div>
             <div type="page" n="2"> </div>
```
Comment (whanley) [inline 1907-07-01.xml]: Move the `</div>` tag, which ends page 2, to the end of page 2, i.e. just before page 3 starts.

File: 1907-07-01.xml [hunk 8]
```diff
@@ -1170,11 +1170,404 @@
...
+                        </row>
+                        <row>
+                            <cell>amassol</cell>
+                            <cell><measure>753.9</measure></cell>
+                            <cell>Almost calm</cell>
+                            <cell><measure unit="dC">31</measure></cell>
+                            <cell>Very slight</cell>
+                        </row>
+                    </table>
+                </div>
```
Comment (whanley) [inline 1907-07-01.xml]: You need another `</div>` to close the weather section

**Comment by whanley** [comment]
Hannah, your Tuesday issue is maybe not the latest version? It's missing lots of material. Others seem pretty full.

**Comment by hcf16b** [comment]
I am at work but I will check as soon as possible (by tonight) and fix it if needed.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by whanley** [comment]
Okay, I'll merge this. Send a new pull request if you find the full file.

**Comment by whanley** [comment]
This is the file to compare your version with: https://github.com/dig-eg-gaz/content/blob/master/1907-07-02.xml

**Comment by hcf16b** [comment]
Good afternoon, I am mostly done with my analysis project but I have been really struggling to find other people’s projects. I don’t know where to locate their projects to compare them to mine.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by whanley** [comment]
https://dig-eg-gaz.github.io/analysis/analysis

**Pull request merged**